src/middlewared/middlewared/plugins/pool.py

```diff
@@ -287,5 +288,15 @@ def configure_resilver_priority(self):
             self.dismissed_import_disk_jobs.add(current_import_job["id"])
 
 
+async def _handle_zfs_events(middleware, event_type, args):
+    data = args['data']
+    if data.get('type') == 'misc.fs.zfs.scrub_finish':
+        await middleware.call('mail.send', {
+            'subject': f'{socket.gethostname()}: scrub finished',
```

Comment:
I think maybe we should create some high-level function for sending E-mails? So it
a) Insert hostname into subject automatically
b) Add some pretty HTML template for text

Reply:
Sounds like a good idea, we use that in quite a few places. Do you want to create a ticket for yourself to implement that?

Reply:
@william-gr yes, I'll do it now